src/mui/list/List.js

```diff
@@ -102,10 +106,9 @@ export class List extends Component {
         return this.props.location.pathname;
     }
 
-    refresh = (event) => {
+    refresh = event => {
         event.stopPropagation();
-        this.fullRefresh = true;
-        this.updateData();
```

Comment:
I'm a bit concerned that you remove this. In my experience, it was necessary to update data here, or race conditions would occur, and the list wouldn't fully refresh. This is because you probably refresh references *before* refreshing the list.